### PR TITLE
MiniMap tooltips on hover

### PR DESCRIPTION
**Pull request by nansonzheng**
Small addition requested by Ryan. Adds tooltips displaying an NPC's name or a portal's destination name when their corresponding markers are hovered over on the MiniMap.

### Changes to files outside MiniMap.h/.cpp  
- **TextTooltip.cpp**: Added condition to shrink frame cover by half to make it fit a one-liner tooltip; centered text label on one-liner tooltips

### Notes  
- Current official client also displays a list of NPCs on the next map on portal marker hover, and NPC functions on NPC marker hover. They both require a line separator which isn't implemented in Texttooltip yet, and portal tooltip requires icons that I haven't found yet.

### CODE REVIEW

**Comment by nansonzheng** [comment]
For some reason, the PR also contains code from the NPC list.

File: IO/UITypes/UIMiniMap.cpp [hunk 1]
```diff
@@ -54,7 +54,7 @@ namespace ms
 		player_marker = Animation(marker["user"]);
 		selected_marker = Animation(MiniMap["iconNpc"]);
 
-		position.shift_y(-Constants::VIEWYOFFSET);
+		//position.shift_y(-Constants::VIEWYOFFSET);
```
Comment (ryantpayton) [inline IO/UITypes/UIMiniMap.cpp]: Uncomment this

File: IO/UITypes/UIMiniMap.cpp [hunk 2]
```diff
@@ -189,8 +202,49 @@ namespace ms
...
+		}
+		
+		if (!found)
+		{
+			for (auto sprite : static_marker_info)
+			{
+				Rectangle<int16_t> marker_spot(sprite.second, sprite.second + 8);
+				if (type == Type::MAX)
+					marker_spot.shift(Point<int16_t>(0, max_adj));
+				std::cout << "Cursor: " << cursor_relative.to_string() << std::flush << "\r";
```
Comment (ryantpayton) [inline IO/UITypes/UIMiniMap.cpp]: Remove this line

**Comment by nansonzheng** [comment]
No problem, I should be sorry for making you catch these minor things for me!

**Pull request merged**